client/src/components/Ticketing/Icons.tsx

```diff
@@ -215,6 +215,67 @@ const XIcon = (props: IconProps) => {
   );
 };
 
+const CirclePlusIcon = (props: {className?: string}) => {
```

Comment:
Do you think you could structure the moved Icons similar to the others, to use the `IconProps`, have default values for `className` and `strokeWidth`, and then alphabetize them.

Reply:
Fixed